.eslintignore

```diff
@@ -1,3 +1,4 @@
 build/
 coverage/
 node_modules/
+src/registerServiceWorker.js
```

Comment:
Why is this file being ignored?

Reply:
There are a bunch of errors and I didn't really know how to resolve them